src/CONST.ts

```diff
@@ -507,6 +507,7 @@ const CONST = {
         COMBINED_TRACK_SUBMIT: 'combinedTrackSubmit',
         CATEGORY_AND_TAG_APPROVERS: 'categoryAndTagApprovers',
         NEW_DOT_QBD: 'quickbooksDesktopOnNewDot',
+        NEW_TRAVEL_PROVISIONING: 'newTravelProvisioning',
```

Comment:
I know that this isn't in review yet, but I realized that I made a mistake on the main issue, this should be NEW_DOT_SPOTNANA_TRAVEL: 'newDotSpotnanaTravel';